src/Accounting/index.ts

```diff
@@ -0,0 +1,202 @@
+import { FlatOrder } from "../Types/accounting";
+
+const RYAN_6 = 6; // a number accredited to our good friend Ryan Garner
+const LIQUIDATION_GAS_COST = 25; // When gas price is 250 gwei and eth price is 1700, the liquidation gas cost is 25 USD.
```

Comment:
What is the use-case of this file? Is it going to be in production? if so, is there any way to query this?